components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -621,9 +624,10 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         }
 
         // A session is required to continue with an activated onboarding flow.
-        if (this.showOnboardingFlowCondition?.value) {
-            throw new ResponseError(ErrorCodes.ONBOARDING_IN_PROGRESS, "Dedicated Onboarding Flow in progress.");
-        }
+        // TODO(gpl): Re-enable once we have an onboarding setup view
+        // if (this.showOnboardingFlowCondition?.value) {
+        //     throw new ResponseError(ErrorCodes.ONBOARDING_IN_PROGRESS, "Dedicated Onboarding Flow in progress.");
```

Comment:
@geropl did you try to just comment out the line, leaving the rest in place? The effect would be to not trigger `ONBOARDING_IN_PROGRESS`, which is what I read from the issue. 

That would also be a good change to validate that feature flags do work in the cell, no?

Reply:
Why? If `enableDedicatedOnboardingFlow` FF is set _and_ this line is disabled, the preceding `SETUP_REQUIRED` is not thrown, see. `&& !this.showOnboardingFlowCondition?.value`

Reply:
That only will work if FFs are working in the cell, which as I understood, is already the case.

Reply:
Moving forward to get unblocked here :+1: